### ISSUE TITLE
AUDIT - Menu Bar Zoom

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyterlab-accessible-themes#11]
- [ ] 200%
- [ ] 400%
- [ ] Change the default value of text size from 16 to 30 in the web browser
- [ ] Test with different screen sizes

## References

- [Theme scope checklist](https://github.com/isabela-pf/accessibility/pull/1/files#diff-084cbb6818caeb52700399405bb8c8e8ed366445ad27c7ac83c70aa10f519382)
- [Audit spreadsheet](https://docs.google.com/spreadsheets/d/1yaWY--2zBi7KqPAnMMbJp_aVXFFT8sBDdci8UfpKjO4/edit#gid=0)

**Comment by steff456** [comment]
These are the results of the Zoom Audit,

## Icon behavior

<img width="1216" alt="image" src="https://user-images.githubusercontent.com/20992645/182727119-ee375776-1572-44a3-99ce-b70f62b76fb4.png">

## Menubar behavior

<img width="1215" alt="image" src="https://user-images.githubusercontent.com/20992645/182727190-5424d41e-20f3-48cf-be9d-c10cd1f51f42.png">

## Notes

- The height is defined in px, therefore it doesn't change with the zoom or with the default text size defined in the web browser
- The menu items start getting lost as zoom increases
- The dropdown menus are not visible or usable with 400% zoom
- Changing the default text size doesn't do any effect in Jlab UI, but changing the minimum text size in the browser does create changes where text overflows.
- Modifying to responsive layouts break the sidebar and its usability, in the same way it breaks in 400% zoom

## Suggestions

- The definition of height is working as they are and they have sense to leave them in px.
- A hamburger menu may need to be added when the space is smaller to avoid loosing items. This needs to be discussed.
- The dropdown menus should open to the left/right depending where there's more space in the screen.
